bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/internal/ComponentChannelType.java

```diff
@@ -26,11 +26,14 @@ public enum ComponentChannelType {
     COLOR("ha-color"),
     DIMMER("ha-dimmer"),
     IMAGE("ha-image"),
+    LOCATION("ha-location"),
     NUMBER("ha-number"),
     ROLLERSHUTTER("ha-rollershutter"),
     STRING("ha-string"),
     SWITCH("ha-switch"),
-    TRIGGER("ha-trigger");
+    TRIGGER("ha-trigger"),
+
```

Comment:
```suggestion
```

Reply:
I assume you resolved this because the comma is intentional? (I separate the GPS accuracy channel by an empty line because it's not a "generic" channel like the rest)